akka-http-core/src/main/scala/akka/http/impl/engine/ws/Websocket.scala

```diff
@@ -172,13 +172,12 @@ private[http] object Websocket {
           }
         }
       })
-      val pullIn = () ⇒ pull(in)
+      val pullIn = () ⇒ tryPull(in)
```

Comment:
Hm, try pull is `if (!closed(in))`, I thought we'll want to pull `if (!hasBeenPulled(in))` - why try here?